diff --git a/src/tint/lang/wgsl/reader/BUILD.cmake b/src/tint/lang/wgsl/reader/BUILD.cmake
index 2d94eb5..eac98d1 100644
--- a/src/tint/lang/wgsl/reader/BUILD.cmake
+++ b/src/tint/lang/wgsl/reader/BUILD.cmake
@@ -45,6 +45,7 @@ tint_target_add_dependencies(tint_lang_wgsl_reader lib
   tint_lang_wgsl_program
   tint_lang_wgsl_reader_lower
   tint_lang_wgsl_reader_parser
+  tint_lang_wgsl_inspector
   tint_lang_wgsl_reader_program_to_ir
   tint_lang_wgsl_resolver
   tint_lang_wgsl_sem
diff --git a/third_party/CMakeLists.txt b/third_party/CMakeLists.txt
index e8b1165..88af692 100644
--- a/third_party/CMakeLists.txt
+++ b/third_party/CMakeLists.txt
@@ -27,7 +27,7 @@ if (NOT TARGET SPIRV-Headers)
     set(SPIRV_HEADERS_SKIP_INSTALL ON CACHE BOOL "" FORCE)
 
     message(STATUS "Tint: using SPIRV-Headers at ${DAWN_SPIRV_HEADERS_DIR}")
-    add_subdirectory(${TINT_SPIRV_HEADERS_DIR} "${CMAKE_CURRENT_BINARY_DIR}/spirv-headers")
+    # add_subdirectory(${TINT_SPIRV_HEADERS_DIR} "${CMAKE_CURRENT_BINARY_DIR}/spirv-headers")
 endif()
 
 # Needs to come before SPIR-V Tools
@@ -52,7 +52,7 @@ if (NOT TARGET SPIRV-Tools)
     endif()
 
     message(STATUS "Tint: using SPIRV-Tools at ${TINT_SPIRV_TOOLS_DIR}")
-    add_subdirectory(${TINT_SPIRV_TOOLS_DIR} "${CMAKE_CURRENT_BINARY_DIR}/spirv-tools" EXCLUDE_FROM_ALL)
+    # add_subdirectory(${TINT_SPIRV_TOOLS_DIR} "${CMAKE_CURRENT_BINARY_DIR}/spirv-tools" EXCLUDE_FROM_ALL)
 endif()
 
 if(NOT TARGET glslang AND ${TINT_BUILD_GLSL_WRITER} AND ${TINT_BUILD_CMD_TOOLS})
